### ISSUE TITLE
Screenreader Issue - Menu Button does not announce state

### ISSUE THREAD

**Issue by James-Green** [w3c/wai-quick-start#293]
In Developing, Keyboard example Menu Button does not announce state so the user doesn’t know if it’s expanded or collapsed.
![screen shot 2015-09-30 at 10 14 41 pm](https://cloud.githubusercontent.com/assets/13589074/10212125/7f75191a-67c0-11e5-8661-9d5cf270e8c0.png)
